init-ci.ps1

```diff
@@ -114,6 +114,15 @@ Write-Host "Preparing your Sitecore Containers environment!" -ForegroundColor Gr
 
 # Check for Sitecore Gallery
 Import-Module PowerShellGet
+
+# Set default SitecoreGallery
```

Comment:
This block is already executed by init.ps1. Do we really need it in init-ci.ps1 as well? init.ps1 will always be run before init-ci.ps1.

Reply:
We haven't been running init.ps1 in the pipeline - only init-ci.ps1. Also when setting up internally (internal NuGet Source) we need to uninstall/re-install the Sitecore Docker Tools to an internal version.